portal_pages/templates/portal_pages/includes/contact.html

```diff
@@ -0,0 +1,44 @@
+
+<div class="third"><div class="cushion">
+
+    <h2>Contact</h2>
+
+    {% if contact.address_1 %}
+        <p>{{ contact.address_1 }}</p>
+    {% endif %}
+
+    {% if contact.address_2 %}
+        <p>{{ contact.address_2 }}</p>
+    {% endif %}
+
+    <p>
```

Comment:
Maybe make this p tag conditional on having city or state? The empty p may make a bit more space than wanted in the case of neither?